releasenotes/25519.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v1
+kind: bug-fix
+area: telemetry
+issue: 25154
+releaseNotes: |
+  Removed the `pilot_xds_eds_instances` and `pilot_xds_eds_all_locality_endpoints` Istiod metrics, which were not
```

Comment:
```suggestion
  *Removed* the `pilot_xds_eds_instances` and `pilot_xds_eds_all_locality_endpoints` Istiod metrics, which were not
```

Reply:
Thanks for adding this @howardjohn One minor suggestion, but other than that looks great to me :)

Reply:
Fixed. Highly recommend best practice for release notes review is to approve w/ a hold, or people will stop adding release notes :slightly_smiling_face:

Reply:
Thanks. I can do that :)